engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobContext.java

```diff
@@ -212,7 +219,12 @@ private ComponentJob getComponentByKey(final AnalysisJob job, final String queri
         return null;
     }
 
-    public String getResultPath() {
+    /**
+     * Gets the path defined in the job properties file
+     * 
+     * @return
+     */
+    public String getResultPathUsercustomized() {
         if (_customProperties != null) {
             if (_customProperties.containsKey(DATA_CLEANER_RESULT_PATH_PROPERTY)) {
```

Comment:
This if-block doesn't archieve anything since the next line would also return null when the key is not represented in the map.